packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -72,15 +72,15 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
       if (createOrganizationMode === 'navigation') {
         return navigateCreateOrganization();
       }
-      openCreateOrganization({ afterCreateOrganizationUrl });
+      return openCreateOrganization({ afterCreateOrganizationUrl });
     };
 
     const handleManageOrganizationClicked = () => {
       close();
       if (organizationProfileMode === 'navigation') {
         return navigateOrganizationProfile();
       }
-      openOrganizationProfile({
+      return openOrganizationProfile({
```

Comment:
🙃 the changes in this file seems like a bug fix! Nice! Can we move it to a different commit?